spec/integration/init_web_spec.cr

```diff
@@ -39,14 +39,22 @@ describe "Initializing a new web project" do
         should_run_successfully "lucky gen.page Users::IndexPage"
         should_run_successfully "lucky gen.component Users::Header"
         should_run_successfully "lucky gen.resource.browser Comment title:String"
+        should_run_successfully "lucky gen.task email.monthly_update"
+        should_run_successfully "lucky gen.secret_key"
 
         File.read("src/actions/comments/index.cr").should contain "Comments::Index"
+        File.read("src/pages/comments/index_page.cr").should contain "Comments::IndexPage"
+        File.read("src/models/comment.cr").should contain "Comment < BaseModel"
+        File.read("src/queries/comment_query.cr").should contain "CommentQuery"
+        File.read("src/operations/save_comment.cr").should contain "SaveComment"
         File.read("src/actions/api/users/show.cr").should_not be_nil
         File.read("src/actions/users/show.cr").should_not be_nil
         File.read("src/pages/users/index_page.cr").should_not be_nil
         File.read("src/components/users/header.cr").should contain "Users::Header < BaseComponent"
-        should_run_successfully "crystal build src/start_server.cr"
```

Comment:
We said build this file, but never actually ran the setup. If you booted this app before, it would have failed with pending migrations and such.